pkgs/applications/editors/hmm/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, lib
+, makeWrapper
+, php
+, ncurses
+, coreutils
+, xclip
+}:
+
+
+stdenv.mkDerivation rec {
+  name = "h-m-m";
+
+  src = builtins.fetchGit {
+    url = "https://github.com/nadrad/h-m-m.git";
+    rev = "ecf74433844e5542f78f64f8a79807818743adff";
+  };
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r h-m-m $out/bin/h-m-m
+  '';
+  postFixup = ''
+    wrapProgram $out/bin/h-m-m \
+      --set PATH ${lib.makeBinPath [
+        php
+        ncurses
+        coreutils
+        xclip
+      ]}
+  '';
+
```

Comment:
You likely need to add yourself to the maintainers list. 

https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix

You should do this in a separate commit.